### PR TITLE
Basic project structure

### PR DESCRIPTION
**Pull request by kate-holdener**
Resolves issue #10 
Basic project structure is initialized. Backend (server) is using flask, frontend (client) is using React JS. Frontend successfully talks to the backend, retrieving dummy data about scheduled shifts.

### CODE REVIEW

**Comment by loganwyas** [comment]
Ran this on Windows 10 and it works. The README for the server may need to be adjusted, though, as the command for activating the virtual environment is different between Windows and MacOS

File: client_app/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
```
Comment (AbhilashKotha) [inline client_app/README.md:8]: We may need to add "npm install" command for first-time users

**Review by AbhilashKotha — CHANGES_REQUESTED** [review]
@kate-holdener The app structure works perfectly for me. I am able to run both server and client, able to fetch the data as well. Just one suggestion is to add "npm install" for first time users.

File: client_app/README.md (hunk 1 above)
Comment (kate-holdener) [reply client_app/README.md:8]: Thank you @AbhilashKotha. I updated the README.md file with the instruction to run "npm install"

Comment (AbhilashKotha) [reply client_app/README.md:8]: I think we can Merge this now. Once we have this folder structure merged, we can commit our files to the wireframes directory.

**Review by loganwyas — COMMENTED** [review]
I personally think that it is good to be merged in.

File: client_app/README.md (hunk 1 above)
Comment (kate-holdener) [reply client_app/README.md:8]: For the sake of following the process, can you approve the PR?

**Review by AbhilashKotha — APPROVED** [review]
lgtm;

**Pull request merged**